lib/before.js

```diff
@@ -62,8 +62,8 @@ module.exports = function(scope, cb) {
 	if (invalidPackageJSON) {
 		return cb.invalid('Sorry, this command can only be used in the root directory of a Sails project.');
 	}
-
-	if (!scope.id) {
+	// Validate controller name
+	if (!scope.id || scope.id[scope.id.length - 1] === '/') {
```

Comment:
Hey @1001v!  How's about we go ahead and bump this package to using lodash@3.10.1?

Then you can do:

```js
if (!scope.id || _.endsWith(scope.id, '/')) {
  return cb.invalid('Usage: sails generate controller <controllername> [action ...]');
}
```